TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -12,7 +12,11 @@
 #ifdef CARTHAGE
 #import <TradeItIosTicketSDK2Carthage/TradeItIosTicketSDK2Carthage-Swift.h>
 #else
-#import <TradeItIosTicketSDK2/TradeItIosTicketSDK2-Swift.h>
```

Comment:
We can't use `@import TradeItIosTicketSDK2.Swift;` since we are inside the module. This is a workaround for the way Cocopod build static libs and we might be able to remove in the future if it was fixed on their side.